### ISSUE TITLE
Vue UI loads with Blank screen

### ISSUE THREAD

**Issue by mikebchu** [vuejs/vue-cli#5353]
### Version
4.2.3

### Environment info
```
MacOS Catalina, Chrome/Safari, Firefox, 
```

### Steps to reproduce
"vue ui" in command line

### What is expected?
Vue UI dashbord

### What is actually happening?
Blank screen

---
Running on localhost:8000

**Comment by jd14** [comment]
I have the same problem, did you find a solution?

**Comment by mikebchu** [comment]
No, I tried just about everything before switching back to ReactJS.

Odd that are no online posts with the same issue.

Maybe we are the only 2 who have experienced this issue.
